test/unit/test-lambda.js

```diff
@@ -863,28 +893,38 @@ describe('Lambda Functions', async() => {
             publishLambdaVersionCounter = 0;
```

Comment:
Same here (counters reset). Also the variables above are already defined, right?

Reply:
Sorry for the wait, I will have a look this evening and push changes as needed.

Reply:
Done.